content/en/blog/_posts/2022-12-06-non-graceful-node-shutdown-to-beta.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Non-Graceful Node Shutdown Moves to Beta"
+date: 2022-12-06T10:00:00-08:00
```

Comment:
Please fix this, along with the path to the Markdown file.

Reply:
Done